src/excel/Workbook.d.ts

```diff
@@ -42,6 +42,16 @@ export declare interface Workbook {
    */
   table(name: string): Table;
 
+  /**
+   * Add a table to a workbook. Note, that it is not possible
+   * to define the name at this point, one has to rename the
+   * table afterwards (see Table#rename).
+   *
+   * @param address address of table, e.g. Sheet1!A1:C1
+   * @param hasHeaders whether the table has headers
+   */
+  addTable(address, hasHeaders): Promise<Table>;
```

Comment:
what about adding an implicit `rename` after creation, if a name is given?
this should also check if the table with the given name already exists.

Reply:
Nice one 💡

Reply:
Please re-verify